ccd-definition/FixedLists/CareSupervision/manage-documents/ManageDocumentTypeListLA.json

```diff
@@ -9,8 +9,8 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "ManageDocumentTypeListLA",
-    "ListElementCode": "COURT_BUNDLE",
-    "ListElement": "Court bundle",
+    "ListElementCode": "HEARING_DOCUMENTS",
```

Comment:
again not sure if we need to be careful here with deprecation

Reply:
I think no, here is just changing the value of the drop down list instead of the whole field. And I can't search any ListElementCode in DEPRECATED.json